llvm/tools/sycl-post-link/sycl-post-link.cpp

```diff
@@ -751,7 +794,7 @@ bool isTargetCompatibleWithModule(const std::string &Target,
 }
 
 std::vector<std::unique_ptr<util::SimpleTable>>
-processInputModule(std::unique_ptr<Module> M) {
+processInputModule(std::unique_ptr<Module> M, LLVMContext &Context) {
```

Comment:
No need for `LLVMContext` here.

Reply:
Hi, @maksimsab 
Done.
Thanks for the careful review.